karapace/protobuf/io.py

```diff
@@ -274,14 +281,18 @@ def writer_mp(
     # To avoid problem with enum values for basic SerDe support we
     # will isolate work with call protobuf libraries in child process.
     if __name__ == "karapace.protobuf.io":
-        queue: _WriterQueue = Queue()
-        p = Process(target=writer_process, args=(queue, config, writer_schema, message_name, datum))
+        writer_queue: _WriterQueue = Queue(1)
```

Comment:
why only 1 message in the queue?

Reply:
does it make sense to proceed with just 1 at a time?

Reply:
There is nothing else put to queue than one message at a time which the caller waits for. I decided just to make the size explicit here.

Reply:
didn't notice wasn't reused and we have 1 process for each message